prometheus/prometheus.yml

```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval: 5s
+  scrape_timeout: 5s
```

Comment:
At least for production, 1 minute should be enough.

Reply:
Thanks, I see.  So we should include two different configuration files.  I'll add one.

You presumably mean that `scrape_interval` should be set to 1 minute, correct?  Or should it be set for `scrape_timeout` as well?

Reply:
I would suggest to remove the values, so it uses the defaults.

scrape_interval 1m
scrape_timeout 10s